### PR TITLE
Feature/mvp application

### PR DESCRIPTION
**Pull request by ahmed-ae**
Various fixes for MVP application

## Description
Few fixes for the MVP application

## Motivation
-  Back button
- Saving correct year
- Filling drop downs

## How Has This Been Tested?
Been able to submit an application sucessfully

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [x] I have read the Contributing guide.
- [x] My code follows the code style of this project.
- [x] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
+TELERIK_ENCRYPTION_KEY=j7IR`bB$#Fn:!iZ@[okN2#N\oWT\[Y<FX0]MGqSWJx<D]p$~wY9#;hO1Pz$bz()u[Aw{q&R3ZM4(t>;EGG*cR]%`&4KiP6#T2;z*MwRX#8c4>RAd({qrTV)9S1u)]6Ys
```
Comment (robearlam) [inline .env:17]: Please remove these values.

File: .env [hunk 2]
```diff
@@ -65,8 +65,8 @@ PeopleGraphQL_UrlEdit=http://cd/api/content/people
 #            Okta - can inject environment vars             #
 #               DO NOT CHECK IN REAL ACCOUNTS               #
 #############################################################
-OKTA_DOMAIN=
-OKTA_CLIENT_ID=
-OKTA_CLIENT_SECRET=
+OKTA_DOMAIN=https://oktapreview.sitecore.com
+OKTA_CLIENT_ID=0oa11wt68a2ALFUB20h8
+OKTA_CLIENT_SECRET=_h32pmlHl9xeZjwnHK02lg9UVgytHi7GC45adgrm
```
Comment (robearlam) [inline .env:70]: Please remove these values.

**Review by robearlam — CHANGES_REQUESTED** [review]
A couple of comments that need fixing with the .env file again.

Also, you've added `mkcert.exe` into the repo at the following location, can this please be removed: `docker/traefik/certs/mkcert.exe`

File: .env (hunk 1 above)
Comment (ahmed-ae) [reply .env]: Removed

File: .env (hunk 2 above)
Comment (ahmed-ae) [reply .env]: removed

**Review by robearlam — APPROVED** [review]

**Pull request merged**